pages/common/tmux.md

```diff
@@ -19,18 +19,41 @@
 
 `tmux attach`
 
-- Detach from the current session (inside a tmux session):
+INSIDE A TMUX SESSION:
+
+- Detach from the current session:
 
 `Ctrl-B d`
 
-- Create a new window (inside a tmux session):
+- Terminology (from `man tmux`):
+  -   Session: collection of pseudo terminals managed by tmux
+  -   Window: occupies the entire screen, can be split into panes (rectangles)
+  -   Pane: one pseudo terminal
+
+- Create a new window:
 
 `Ctrl-B c`
 
-- Switch between sessions and windows (inside a tmux session):
+- Cycle through windows:
+
```

Comment:
```suggestion
- While running tmux, cycle through windows:

```
please follow the syntax used here.

I have never used tmux, does it use caps for its name, like `TMux`

Reply:
> please follow the syntax used here.
> 
> I have never used tmux, does it use caps for its name, like `TMux`

I have always seen it as `tmux`

Reply:
> I have never used tmux, does it use caps for its name, like `TMux`

No, it's just `tmux`

Reply:
Ok